src/components/Categories.jsx

```diff
@@ -0,0 +1,4 @@
+import styles from '../styles/Books.module.css';
+
+const Categories = () => <div className={styles.BOOKS}>Categories</div>;
```

Comment:
- [ ] The `Categories` component should include a `button` with the text "Check status", kindly consider creating a button here so the user has a way to check the status (for now the button will not have any functionality)

Reply:
Thanks Ismailco I applied changes

Reply:
Thanks, Ismailco I applied the changes